Cargo.toml

```diff
@@ -8,16 +8,23 @@ description = "NewTek NDI Plugin"
 
 [dependencies]
 glib = { version = "0.8.0", features = ["subclassing"] }
-gstreamer = { version = "0.14.0", features = ["subclassing"] }
+gobject-sys = "0.9"
+gstreamer = { version = "0.14.3", features = ["subclassing", "v1_12"] }
 gstreamer-base = { version = "0.14.0", features = ["subclassing"] }
 gstreamer-audio = "0.14.0"
-gstreamer-video = "0.14.0"
+gstreamer-video = "0.14.3"
```

Comment:
```suggestion
gstreamer-video = { version = "0.14.3", features = ["v1_12"] }
```

Using ubuntu 18.04 (gstreamer version 1.14.4-1~ubuntu18.04.1) and cargo 1.36.0 (c4fcfb725 2019-05-15). 


<details>
 <summary>Error:</summary>

```
$ cargo build --no-default-features
...
   Compiling gst-plugin-ndi v1.0.0 (/home/rubenrua/src/github.com/teltek/gst-plugin-ndi)
error[E0433]: failed to resolve: could not find `VideoFieldOrder` in `gst_video`
    --> src/receiver.rs:1339:58
     |
1339 |                 builder = builder.field_order(gst_video::VideoFieldOrder::TopFieldFirst);
     |                                                          ^^^^^^^^^^^^^^^ could not find `VideoFieldOrder` in `gst_video`

error[E0599]: no method named `field_order` found for type `gst_video::VideoInfoBuilder<'_>` in the current scope
    --> src/receiver.rs:1339:35
     |
1339 |                 builder = builder.field_order(gst_video::VideoFieldOrder::TopFieldFirst);
     |                                   ^^^^^^^^^^^

```
</details>